adoc/code/attributes.cpp

```diff
@@ -1,19 +1,27 @@
 // Copyright (c) 2011-2021 The Khronos Group, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-// Kernel defined as a lambda
+// Kernel defined as a lambda.
```

Comment:
I do not think we use `.` at the end of single-sentence comments in the specification.

Reply:
Fixed here and below.